packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -479,15 +479,15 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
                         {travelMode === travelModes.DRIVING && <DriveIcon />}
                         {travelMode === travelModes.BICYCLING && <BikeIcon />}
                         <div>{t('Distance')}:</div>
-                        <div className="wayfinding__meters">{totalDistance && <mi-distance unit={distanceUnitSystem} meters={totalDistance} />}</div>
+                        <div className="wayfinding__meters">{totalDistance && isDirectionReady && <mi-distance unit={distanceUnitSystem} meters={totalDistance} />}</div>
```

Comment:
Is this change necessary? The `<mi-distance>` seems to be showing nothing anyway.

Reply:
It does change when you f. ex. switch directionsTo to Viborg. Without it, for a few seconds the time/distance is shown from the previous search. It's a lil-UX improvement.

Reply:
👏